[2024-10-30T21:48:04.232+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-30T21:48:04.249+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_setup_teardown.random_error manual__2024-10-30T21:47:58.433442+00:00 [queued]>
[2024-10-30T21:48:04.256+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_setup_teardown.random_error manual__2024-10-30T21:47:58.433442+00:00 [queued]>
[2024-10-30T21:48:04.257+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-30T21:48:04.269+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): random_error> on 2024-10-30 21:47:58.433442+00:00
[2024-10-30T21:48:04.275+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1067) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-30T21:48:04.277+0000] {standard_task_runner.py:72} INFO - Started process 1080 to run task
[2024-10-30T21:48:04.278+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_setup_teardown', 'random_error', 'manual__2024-10-30T21:47:58.433442+00:00', '--job-id', '477', '--raw', '--subdir', 'DAGS_FOLDER/lesson_6_setup_teardown.py', '--cfg-path', '/tmp/tmp29knaa4z']
[2024-10-30T21:48:04.279+0000] {standard_task_runner.py:105} INFO - Job 477: Subtask random_error
[2024-10-30T21:48:04.325+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_setup_teardown.random_error manual__2024-10-30T21:47:58.433442+00:00 [running]> on host 1da8eee663cb
[2024-10-30T21:48:04.398+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_setup_teardown' AIRFLOW_CTX_TASK_ID='random_error' AIRFLOW_CTX_EXECUTION_DATE='2024-10-30T21:47:58.433442+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-30T21:47:58.433442+00:00'
[2024-10-30T21:48:04.400+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-30T21:48:04.400+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-30T21:48:04.401+0000] {logging_mixin.py:190} INFO - Current task name:random_error state:running start_date:2024-10-30 21:48:04.249932+00:00
[2024-10-30T21:48:04.402+0000] {logging_mixin.py:190} INFO - Dag name:dag_setup_teardown and current dag run status:running
[2024-10-30T21:48:04.403+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-30T21:48:04.404+0000] {python.py:240} INFO - Done. Returned value was: 0
[2024-10-30T21:48:04.430+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-30T21:48:04.432+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_setup_teardown, task_id=random_error, run_id=manual__2024-10-30T21:47:58.433442+00:00, execution_date=20241030T214758, start_date=20241030T214804, end_date=20241030T214804
[2024-10-30T21:48:04.441+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-30T21:48:04.443+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-30T21:48:04.443+0000] {logging_mixin.py:190} INFO - Dag name:dag_setup_teardown queued_at:2024-10-30 21:47:58.459269+00:00
[2024-10-30T21:48:04.444+0000] {logging_mixin.py:190} INFO - Task hostname:1da8eee663cb operator:PythonOperator
[2024-10-30T21:48:04.453+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-30T21:48:04.476+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-30T21:48:04.478+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
